searx/settings.yml

```diff
@@ -790,6 +790,23 @@ engines:
     shortcut : nt
     disabled : True
 
+  - name : mankier
+    engine : json_engine
+    paging : True
```

Comment:
You enabled `paging`, yet I cannot see `{pageno}` in the `search_url`. Either add it to the URL or remove `paging: True`.

Reply:
Oops, missed that in whatever engine I copy/pasted from; I've removed paging from the engine definition.